### ISSUE TITLE
How to configure with existing setupTests.js

### ISSUE THREAD

**Issue by fljmayer** [mattphillips/jest-expect-message#11]
This is probably just a documentation issue. I am on Jest 23, and I already have
```
"jest": {
    "setupTestFrameworkScriptFile": "<rootDir>src/react/setupTests.js"
```
Can I somehow initialize `jest-expect-message` within my `setupTests.js` file?

**Comment by saifulss** [comment]
Leaving comment here to help future travellers.

**TL;DR:**
Add `import 'jest-expect-message';` to your `setupTests.ts` file.

**Long version:**
I rolled a fresh `create-react-app` with TypeScript app, which of course came with the `setupTests.ts` file. Note the file extension `.ts`. I installed this `jest-expect-message` dep together with `@types/jest-expect-message`. I added the above-mentioned import statement. Voila! It just works.

Though this is for `.ts`, I expect it to also just work with `.js`. Of course with JS you won't have to bother with installing the `@types` dep.

Hope this helps others.
